src/components/Card.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "styled-components";
+
+import Button from "../components/Button";
+
+const Card = ({ img, title, text, creator, buttonText, onClick }) => {
+  return (
+    <CardContainer>
+      {img && <CardImg src={img} alt={title} />}
```

Comment:
I would say give the default `alt` tag (so the title prop) an empty string because `title` is also necessary here. If the `img` prop is set and not the `title tag you would get `undefined` in the alt. Better to have at least something.